packages/block-editor/src/components/image-size-control/style.scss

```diff
@@ -1,26 +1,3 @@
 .block-editor-image-size-control {
 	margin-bottom: 1em;
-
-	.block-editor-image-size-control__row {
-		display: flex;
-		justify-content: space-between;
-
-		.block-editor-image-size-control__width,
-		.block-editor-image-size-control__height {
-			margin-bottom: 0.5em;
-
-			// Fix the text and placeholder text being misaligned in Safari
```

Comment:
I removed CSS that was added three years ago for a Safari fix, but when testing Safari with the removed code, it looked the same as other browsers.